MembershipTypesAndPriceSetsData.json

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
I like the idea of using json for this however, would it be possible to separate JSON files for different membership type and price set? For example. 

FixedPeriodMembeshipTypeData.json
RollingPeriodMembershipTypData.json
PriceSetsWithFixedPeriodMembershipType.json
PriceSetsWithRollingPeriodMembershipType.json

Reply:
I decided to put the sample data in one place to make it easier for adding and modifying.